docs/http.rst

```diff
@@ -35,8 +35,8 @@ The following section describes how remote servers can send Push
 Notifications to apps running on remote User Agents.
 
 API methods requiring Authorization must provide the Authorization
```

Comment:
Maybe this would be better under the "Push Service Bridge HTTP Interface" section? Placing it here at the top is a little confusing because the "Push Service HTTP API" uses VAPID, not the secret.